website/static/js/pages/sharing-page.js

```diff
@@ -24,6 +29,48 @@ $('body').on('nodeLoad', function(event, data) {
     }
 });
 
+$('.filters').filters({
+    items: ['.contrib', '.admin'],
+    callback: function (filtered, empty) {
+        if ($.inArray('.contrib', empty) > -1) {
+            $('#noContributors').show();
```

Comment:
If you move all this logic to the ViewModel, your callback will be able to update the ViewModel's observables.